client/apis/recipes.js

```diff
@@ -0,0 +1,21 @@
+import request from 'superagent'
+
+// const rootUrl = '/api/v1'
+
+// TODO: create .env file to store my KEY
+// Adjust config/setup -- Dotenv
+
+export function getRecipes(userInput) {
+  return request
```

Comment:
Add a catch incase something does wrong, since you don't control this api this is even more important for you to take into consideration if something goes wrong on their end

Reply:
Converted to async function and wrapped in try/catch.